### PR TITLE
Placeholder Part II: For commenting about install

### PR DESCRIPTION
**Pull request by moxiegirl**
The experimental section in support of #13023 now exists.  This is a placeholder so people can easily see the intended content for the Overview.  Pending are the installation instructions.

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

File: docs/sources/experimental/experimental.md [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
 
-The information below describes each feature and the Github pull requests and issues associated with it. If necessary, links are provided to additional documentation on an issue.  As an active Docker user and community member, please feel free to provide any feedback on these features you wish.
+The information below describes each feature and the Github pull requests and
+issues associated with it. If necessary, links are provided to additional
+documentation on an issue.  As an active Docker user and community member,
+please feel free to provide any feedback on these features you wish.
+
+## Install Docker experimental 
+
+TODO: Will experimental be available on all platforms or just some?
```
Comment (thaJeztah) [inline docs/sources/experimental/experimental.md]: I can imagine that depends on the feature, but should be mentioned in the description.

**Comment by thaJeztah** [comment]
LGTM (so far)

Are you planning on updating this PR, or add the info in a follow-up?

File: docs/sources/experimental/experimental.md (hunk 1 above)
Comment (moxiegirl) [inline docs/sources/experimental/experimental.md]: Yes, it is unclear for sure.  I was more thinking boot2docker env versus others...if a feature can only have an effect in one environment, I think that is rare...

**Comment by moxiegirl** [comment]
I'll update this PR with the install. I figure experimental feature developers now have a basis to add their content without relying on the previous PR to close.

**Comment by moxiegirl** [comment]
ping @icecrime @calavera Updated and rebased with plugin material

File: docs/sources/experimental/plugin_api.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ This page is intended for people who want to develop their own Docker plugin.
 If you just want to learn about or use Docker plugins, look
 [here](/userguide/plugins).
 
+This is an experimental feature. For information on installing and using experimental features, see [the experimental feature overview](e
```
Comment (thaJeztah) [inline docs/sources/experimental/plugin_api.md]: Looks like something got cut off here

File: docs/sources/experimental/plugins_volume.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
 
-# Overview
+# Experimental: Docker volume plugins
 
 Docker volume plugins enable Docker deployments to be integrated with external
 storage systems, such as Amazon EBS, and enable data volumes to persist beyond
 the lifetime of a single Docker host. See the [plugin documentation](/experimental/plugins)
 for more information.
 
+This is an experimental feature. For information on installing and using experimental features, see [the experimental feature overview](e
```
Comment (thaJeztah) [inline docs/sources/experimental/plugins_volume.md]: Same here (cut off)

**Comment by calavera** [comment]
This needs rebase :pensive:

**Comment by moxiegirl** [comment]
done.  I also need to make the install change. Give me a sec.

**Comment by moxiegirl** [comment]
Author LGTM.  I'm going to merge this when the builds go green. @jfrazelle can you give a quick once over to the install section and make sure I have that correct.

**Comment by thaJeztah** [comment]
LGTM, but obviously, the experimental domain is not yet up.

**Pull request merged**